src/blocks/zesty/Hero/SingleRowHero.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * MUI Imports
+ * */
+
+import React from 'react';
+import { Box, Button, Container, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+/**
+ *  Components Imports
+ * */
+import ArrowRightAltIcon from '@mui/icons-material/ArrowRightAlt';
+import FillerContent from 'components/globals/FillerContent';
+import ZestyImage from 'blocks/Image/ZestyImage';
+
+const TwoRowsHero = ({
+  eyebrow = FillerContent.header,
+  header = FillerContent.header,
+  description = FillerContent.description,
+  cta_left = FillerContent.cta,
+  cta_left_link = FillerContent.href,
+  cta_right = FillerContent.cta,
+  cta_right_link = FillerContent.href,
+  isMobile,
+  image
+}) => {
+  const theme = useTheme();
+
+  return (
+    <Box component="section" sx={{ pt: 15, pb:8 }}>
+      <Container
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          flexDirection: 'column',
+        }}
+      >
+        <Box
```

Comment:
we can use `Stack` from mui instead of `Box` [here](https://mui.com/material-ui/react-stack/)